setupTests.js

```diff
@@ -1,18 +1 @@
-import IntlPolyfill from 'intl';
-import 'intl/locale-data/jsonp/en-GB';
 import '@testing-library/jest-dom/extend-expect';
-
-global.console = {
-  log: jest.fn(), // console.log are ignored in tests
-  error: console.error,
-  warn: console.warn,
-  info: console.info,
-  debug: console.debug
-};
```

Comment:
This global.console is still required - nothing to do with dates

Reply:
You'll need to add it back into the shared plan config too

Reply:
yes! Will do!